IBPSA/Fluid/HeatPumps/Validation/BaseClasses/PartialHeatPumpValidation.mo

```diff
@@ -0,0 +1,181 @@
+within IBPSA.Fluid.HeatPumps.Validation.BaseClasses;
+partial model PartialHeatPumpValidation
```

Comment:
1. Please ensure description strings are provided for all variables.
2. I think the `Medium` modifications of each component can be made `final`.